.github/workflows/push-tests.yml.j2

```diff
@@ -1,10 +1,11 @@
-{% if distro_release == "rawhide" %}
 name: Run tests on push
 on:
   push:
     branches:
       - master
       - fedora-[0-9]+
+      - rhel-9
```

Comment:
We actually dont hardcode branch names. You need to generate it. 

See {% for branch in supported_branches ...

Reply:
Or wait on https://github.com/rhinstaller/anaconda/pull/5598 which will add matrix builder. Both options are fine.